docs/upgrade/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: documentation
+title: Katello Upgrade
+sidebar: sidebars/documentation.html
+---
+
+# Katello Upgrade
+
+Katello supports upgrades from version 2.0.  For users transitioning from 1.4, please see - [Transition Guide](/docs/{{ site.version }}/installation/2.0-transition.html).
+
+## Step 1 - Backup
+
+If Katello is running on a Virtual Machine, it would be a good idea to take a snapshot prior to upgrading. Otherwise, take a backup of the relevant databases by following the instructions here (TODO).
+
+## Step 2 - Operating System
+
+Ensure your operating system is fully up-to-date:
+
+```
+# yum -y update
+```
+
+## Step 3 - Repositories
+
+Update the Foreman and Katello release packages:
+
+  * RHEL6 / CentOS 6:
+
+```
+  # yum update -y http://fedorapeople.org/groups/katello/releases/yum/2.1/katello/RHEL/6Server/x86_64/katello-repos-latest.rpm
+  # yum update -y http://yum.theforeman.org/releases/1.7/el6/x86_64/foreman-release.rpm
+```
+
+  * RHEL7 / CentOS 7:
+
+```
+  # yum update -y http://fedorapeople.org/groups/katello/releases/yum/2.1/katello/RHEL/7Server/x86_64/katello-repos-latest.rpm
+  # yum update -y http://yum.theforeman.org/releases/1.7/el7/x86_64/foreman-release.rpm
+```
+
+## Step 4 - Update Packages
+
+Clean the yum cache
+
+```
+# yum clean all
+```
+
+Update/install the required packages packages:
+
+```
+# yum install python-crane qpid-cpp-client-devel
```

Comment:
Definitely. How about here? https://github.com/Katello/katello/blob/master/rubygem-katello.spec#L79  (unless qpid-cpp-client-devel is needed by something Katello is doing and in that case put it in the relevant requires section of that spec file)